css/style.css

```
@@ -16,4 +16,379 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* Variables */
+:root {
+    --orange: #E0633A;
+    --faint-orange: rgba(224, 99, 58, 0.1);
+}
+
+
+/* Accessibility Styles */
+
+.hide-this {
+    visibility: hidden; 
+}
+
+.show-this {
+    visibility: visible;
+}
+
+fieldset {
+    all: unset;
+}
+
+legend.hide-this {
+    all: unset;
+    height: 0;
+}
+
+
+ /* Navigation Styles */
+
+.nav-flex-container {
+    display: flex;
+    justify-content: space-between;
+    height: 4em;
+    width: 80%;
+    margin: auto;
+}
+
+.logo {
+    height: 2.25em;
+    margin: auto 0;
+}
+
+.nav-ul {
+    list-style-type: none;
+    margin: auto 0;
+    padding: 0;
+    font-weight: 450;
+}
+
+.nav-li {
+    display: inline;
+    margin: 0 0.75em;
+
+}
+
+.nav-link {
+    text-decoration: none;
+    color: rgb(165, 165, 165);
+    /* font-size: .90em; */
+}
+
+.nav-link:hover{
+    color: var(--orange);
+}
+
+.meet-karma {
+    font-weight: bold;
+    color: rgb(92, 91, 91);
+}
+
+.active-link {
+    color: var(--orange);
+}
+
+/* Main Styles */
+
+    /* Hero Section Styles */
+
+    .hero-image {
+        background-image: url("../img/first-background.jpg");
+        height: 59vw;
+        background-position: center;
+        background-repeat: no-repeat;
+        background-size: cover;
+        background-position: 37.5% 100%;
+        display: flex;
+        
+    }
+
+    .content-container {
+        margin: auto auto; 
+        /* background-color: darkorange; */
+        width: 100%;
+        text-align: center;
+        padding-bottom: 4em;
+    }
+
+    .hero-text {
+        color: white;
+        font-weight: 300;
+    }
+
+        .hero-h1{
+            font-size: 3em;
+            margin: 0;
+        }
+
+        .hero-h3{
+            font-size: 1.75em;
+            margin: 0.75em 0 1.5em;
+        }
+
+        .hero-btn {
+            background-color: var(--orange);
+            border-radius: 3px;
+            border: 0;   
+            padding: 1em 2em;  
+        }
+
+        .hero-btn-a {
+            text-decoration: none;
+        }
+
+        /* Article Section Styles */
+
+        .article-title {
+            text-align: center;
+            font-weight: 300;
+            font-size: 2.5em;
+            margin-top: 1.5em;
+        }
+
+        .article-text {
+            text-align: center;
+            font-size: 1.5em;
+            margin-top: 1.25em;
+        }
+
+        .article-flex-container{
+            display: flex;
+            justify-content: space-evenly;
+        }
```

At the moment the images are not centered on the page (i.e. the coffee mug is not sitting in the middle of the page as in the reference image). You can set a max-width for the container, and then set a specific width on each article image to make them have the same width and be centered:

```suggestion
        .article-flex-container{
            display: flex;
            justify-content: space-evenly;
            max-width: 56rem;
            margin: 0 auto;
        }

        .article-flex-container article {
            width: 17rem;
        }
```